index.bs

```diff
@@ -7510,6 +7510,15 @@ Note: this extension may be implemented for [=authenticators=] that do not use [
 
         :   <dfn>results</dfn>
         ::  The results of evaluating the PRF for the inputs given in {{AuthenticationExtensionsPRFInputs/eval}} or {{AuthenticationExtensionsPRFInputs/evalByCredential}}. Outputs may not be available during [=registration=]; see comments in {{AuthenticationExtensionsPRFInputs/eval}}.
+
+            Advisement:
```

Comment:
@emlun should this be a note instead?

Reply:
(See also [Bikeshed documentation: Notes, Issues, Examples, Advisements](https://speced.github.io/bikeshed/#notes-etc))